### DISCUSSION TITLE
Icons

### DISCUSSION THREAD

**Discussion by PatriciaAltshuler** [community/community#42347] [Repositories]
### Select Topic Area

Product Feedback

### Body

Hi i am new here a don't even know if i am writing in the right place. I have so many doubts but for know i had a question about icons. I am working with vs code and i am trying to make a website with a slider. The problem is i don't know why my go left and right icons aren't showing. I downloaded them from fonts.google/icons.com or something like that and maybe the file isn't located correctly or i need an extension..? Can someone help.

**Comment by Mod8124** [comment]
Hi, welcome to code :), 
1. there are many pages for icons, I personally choose this page [iconfinder](https://www.iconfinder.com)
2.  look for the icon, and download what you want it, could be `png`, `svg` or `ico(this appears on the top of the page)` be sure that you download in the correct file `inside the same file of the page`
```
> icon-arrow.svg
> index.html
```
3. if the icon it's inside the same file as the page(index.html) you can use `<img src="./icon-arrow.svg">`  or just copy the the `svg` inside the HTML(only SVG)
4.   and that's all also you can use an extension called `image-preview` helps a lot if the icon appears on the left work it xD.

also for the slider check if the icon is outside of the change's content if it's inside it's going to change too.
```html
  <img src="./icon-arrow-left.svg"/>
   <div>this content change</div>
  <img src="./icon-arrow-right.svg"/>
```
 I hope this helps u
